doc/quickstart.rst

```diff
@@ -8,10 +8,10 @@ CadQuery QuickStart
 
 Want a quick glimpse of what CadQuery can do?  This quickstart will demonstrate the basics of CadQuery using a simple example
 
-Prerequisites: Anaconda/Miniconda + CQ-editor running from an environment
+Prerequisites: conda + CQ-editor running from an environment
```

Comment:
Sharing what goes through my head, but couldn't it be possible to setup a [binder](https://mybinder.org/) environment to give quick example of CQ to people ?

Reply:
I think that we tried, but it was really slow

Reply:
Ah ok didn't know, nevermind then